tests/h2integrate/test_all_examples.py

```diff
@@ -126,9 +126,6 @@ def test_wind_h2_opt_example(subtests):
     with subtests.test("Check LCOH"):
         assert model.prob.get_val("financials_group_1.LCOH")[0] < 4.64
```

Comment:
I know this is not part of this PR per se, but It think it would be better to change the 4.64 on the right to the initial LCOH before optimization so that if the models or assumptions change, the test remains valid and useful.

Reply:
Got it, good call, I modified the case and test!